bookstore_api/models.py

```diff
@@ -25,5 +27,5 @@ class UserProfiles(models.Model):
     homeAddress = models.CharField(max_length=100)
     cart = models.JSONField(null=True)
     saveLater = models.JSONField(null=True)
-
+    wishlist = models.JSONField(null=True)
```

Comment:
We have to wait until my merge request is approved and then we can add this 
from django.utils import timezone
and add
wishlist = models.JSONField(null=True)